egs/ami/s5b/RESULTS_mdm

```diff
@@ -64,11 +62,18 @@
 
 
 # cleanup + chain TDNN model, alignments from IHM data (IHM alignments help).
-# local/chain/run_tdnn.sh --mic mdm8 --use-ihm-ali true --stage 12 &
```

Comment:
Actually, I think I'll wait to merge, I'd like you to clean up this RESULTS file.
We are not putting the detailed tuning results in this RESULTS file.  Remove the old results.  We don't  want any detailed comparative results in this file, just the results with and without cleanup, and possibly with and without IHM alignments (but you don't have to update all the results necessarily ).  Don't say "Uses batchnorm components instead of renorm" at this level.